Dockerfile.govuk-base

```diff
@@ -5,11 +5,8 @@
 # Install packages for building ruby
 FROM buildpack-deps
 
-# Install chrome and its dependencies
-RUN apt-get update -qq
-RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb 2>&1 && \
-   apt-get install -y ./google-chrome*.deb && \
-    rm ./google-chrome*.deb
+# Install chromium browser and its webdriver
+RUN apt-get update -qq && apt-get install -y chromium chromium-driver
```

Comment:
Should the same change be applied to the Whitehall and Asset Manager `Dockerfile` as in #576?

Reply:
Nice spot – thanks @ChrisBAshton.